src/stac_utils/google.py

```diff
@@ -22,7 +22,14 @@ def get_credentials(
     scopes: [list[str], str] = None,
     subject: str = None,
 ) -> [service_account.Credentials, None]:
-    """Loads a Google Service Account into a Credentials object with the given scopes"""
+    """Loads a Google Service Account into a Credentials object with the given scopes
+
+    :param service_account_blob: Service account blob
+    :param service_account_env_name: Environmental variable name for service account
+    :param scopes: Desired service account scopes
+    :param subject: Service account subject
+    :return: Service account credentials
```

Comment:
i'd note that you include either the blog or the environment name, and that it defaults to using the SERVICE_ACCOUNT if that's in your os.environ